coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -65,3 +67,29 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
             file.writelines(ignore_diff.modified)
 
         return file_diff_dict
+
+    def get_ignore_comment(self, origin, language):
```

Comment:
Where is the test? I found deprecate_bear and its docstring. Where and how is it run?